sections/04_study.md

```diff
@@ -241,114 +241,61 @@ this was easy to simulate with a computational model. As we learn to better
-[@doi:10.1016/j.tig.2015.05.007]. One study [@doi:10.1101/gr.173518.114] even
-showed that only 33% of predicted regulatory regions could be validated, while a
-class of "weak" predicted enhancers were strong drivers of expression. Yet there
-is growing evidence for their vast ubiquity, making them possibly the
-predominant functional non-coding element. Thus, identifying enhancers is
-critical yet the search space is large.
-
-While previous (non-deep learning) approaches have steadily improved
-promoter prediction, there is little consensus on the best approach and
-performance is poor. Typically algorithms will recognize only half of all
-promoters, with an accompanying high false positive rate
-[@doi:10.1101/gr.7.9.861]. Methods with better sensitivity generally do so at
-the cost of poorer specificity. Conventional identification of enhancers has
-leaned heavily on simple conservation or laborious experimental techniques, with
-only moderate sensitivity and specificity. For example, while chromatin
-accessibility has often been used for identifying enhancers, this also
-"recognizes" a wide variety of other functional elements, like promoters,
-silencers, and repressors.
-
-The complex nature of CREs and our lack of understanding makes them a natural
-candidate for deep learning approaches. Indeed, neural networks were used for
-promoter recognition as early as 1996, albeit with mixed results
-[@doi:10.1016/S0097-8485(96)80015-5]. Since then, there has been much work in
-applying deep learning to this area, although little in the way of comparative
-studies or formal benchmarks. We therefore focus on a few recent important
-studies to outline the state of the art and extant problems.
-
-Basset [@doi:10.1101/gr.200535.115] trained CNNs on DNA accessibility datasets,
-getting a marked improvement on previous methods, albeit still with a high false
-positive rate. The multi-task architecture resembles DeepSEA
-[@tag:Zhou2015_deep_sea], which predicted open chromatin regions and histone
-modifications in addition to TF binding. As noted above, predicting DNA
-accessibility conflates enhancers with other functional sites. Basset also
-featured a useful interpretability approach, introducing known protein binding
-motifs into sequences and measuring the change in predicted accessibility.
-
-Umarov et al. [@doi:10.1371/journal.pone.0171410] demonstrated the use of CNNs
-in recognizing promoter sequences, outperforming conventional methods
-(sensitivity and specificity exceeding 90%). While some results were achieved
-over bacterial promoters (which are considerably simpler in structure), roughly
-similar performance was found for human promoters. This work also included a
-simple method for model interpretation, randomly substituting bases in a
-recognized promoter region, then checking for a change in recognition (see
-Discussion).
-
-Xu et al. [@doi:10.1109/BIBM.2016.7822593] applied CNNs to the detection of
```

Comment:
I still discuss this paper, removed the citation in error. Fixed in michaelmhoffman/deep-review@e5777e978630f66f411fb689bfd999496574867b.